src/components/core/ImageGridItem.tsx

```
@@ -0,0 +1,28 @@
+ * @prop {string} [className] - Additional CSS classes to add to the image.
+ * @returns {JSX.Element} JSX Element for the single image.
+ */
+export function ImageGridItem({
+  img,
+  className,
+}: Omit<ImageGridItemProps, "key">): JSX.Element {
+  return (
+    <div
+      className={mergeClasses("grid__img", className || "")}
+      style={{ backgroundImage: `url(./image/${img})` }}
+    ></div>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)